docs/source/getting-started.md

```diff
@@ -258,17 +272,20 @@ allow running without SSL using the command `jupyterhub --no-ssl`. The
 
 ---
 
-Security is the most important aspect of configuring Jupyter. There are four main aspects of the
-security configuration:
+Security is the most important aspect of configuring Jupyter. These are the
+main elements of a security configuration:
 
 1. SSL encryption (to enable HTTPS)
-2. Cookie secret (a key for encrypting browser cookies)
+2. Cookie secret (a key for encrypting browser cookies) (Beginning in version
+   0.8, cookie secret is deprecated, though available for use.)
```

Comment:
Cookie secret is not deprecated. It is only the shared-cookie behavior that is deprecated. The Hub itself still uses a cookie.